irc_bot_methods.rb

```diff
@@ -0,0 +1,80 @@
+def handle_message_colic(message)
```

Comment:
Don't require a message argument, if you don't use it in the method
```suggestion
def handle_message_colic
```
if you change it like that, don't forget to also change the method calls.

Reply:
lol honestly didn't even realise you could do that 🤣

Reply:
hihi, always good to learn something new 😇